config/environments/production.rb

```diff
@@ -100,4 +100,11 @@
     ExceptionNotification::Rack,
     slack: { webhook_url: 'https://hooks.slack.com/services/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' }
   )
+
+  config.secret_path = '/run/secrets'
+    Raven.configure do |config|
+    config.dsn = "https://#{OperationCode.fetch_secret_with(name: :sentry_credentials)}@sentry.io/147247"
+    config.environments = ['production']
+    config.release = '0e4fdef81448dcfa0e16ecc4433ff3997aa53572'
```

Comment:
Ok. So then according to https://docs.sentry.io/clients/ruby/config/ it will set this automatically. I'm just going to remove the line then.